front/app/containers/MainHeader/NotificationMenu/NotificationMenu.test.tsx

```diff
@@ -1,31 +1,46 @@
-// libraries
 import React from 'react';
-import { shallow } from 'enzyme';
+import { IUserAttributes } from 'services/users';
```

Comment:
Great work on the tests - an awesome start! To have the logic more fully covered, I would add:
1. Test that it displays the empty state when there are no notifications
2. Test that it displays the notification list correctly when there are notifications

Adding these would be ideal, but I also understand if you want to limit the effort on this one, so it's not blocking from my perspective.

Reply:
Ok, I will look into these 👍

Reply:
@IvaKop Started with this but I won't have the time this week. Too much other more urgent work. I've pushed my WIP to a branch and will continue with it in the coming weeks. 

Merging this for now.